src/backend/marsha/core/tests/test_utils_medialive_utils.py

```diff
@@ -124,13 +124,14 @@ def test_get_or_create_input_security_group_matching_tag(self):
 
         self.assertEqual("tagged_security_group", security_group_id)
 
+    @override_settings(AWS_BASE_NAME="test")
```

Comment:
I relealized in the doc this setting is mentioned as not required. But in the code it is if I'm not mistaken? 
https://github.com/openfun/marsha/blob/master/docs/env.md#django_aws_base_name

Reply:
No it's not, every setting class define it's own value: https://github.com/openfun/marsha/blob/1f65ef7e93735afb533b2bde236f6b63e2381747/src/backend/marsha/settings.py#L336